prompting/protocol.py

```diff
@@ -115,6 +115,20 @@ def deserialize(self) -> "Prompting":
         allow_mutation=False,
     )
 
+    messages_hash: str = pydantic.Field(
```

Comment:
do we really need to hash the message? 

cause instead of 

> header: sign(hashed body)
> body: body
> axon verification: hash the body -> check signature VS hashed body 

maybe we can just do the following

> header: sign(body)
> body: body
> axon verification: check signature VS body

Reply:
Hashes are necessary for this scheme to work, unfortunately.  Added documentation for this.